src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/model/EscapeProfile.kt

```diff
@@ -16,9 +16,10 @@ data class EscapeProfile(
     example = "true",
   ) val activeEscapeRisk: Boolean,
 
-  @field:ApiModelProperty(value = "Active escape list alerts") val escapeListAlerts: List<Alert>?,
+  @field:ApiModelProperty(value = "Active escape risk alerts") val escapeRiskAlerts: List<Alert>?,
+
+  @field:ApiModelProperty(value = "Active escape list alerts") val escapeListAlerts: List<Alert>?
 
-  @field:ApiModelProperty(value = "Active escape risk alerts") val escapeRiskAlerts: List<Alert>?
```

Comment:
This restores the original (slightly odd) field order. This is needed to prevent the risk profiler thinking that ALL offenders profiles have changed (just because the json fields appear in a reversed order)